### ISSUE TITLE
Postgres loader: invalid UTF-8 bytes

### ISSUE THREAD

**Issue by abyrd** [conveyal/osm-lib#30]
When running `java -cp target/dependency/*:target/osm-lib-1.2.1-SNAPSHOT.jar com.conveyal.osmlib.PostgresSink http://download.geofabrik.de/europe/germany-latest.osm.pbf "jdbc:postgresql://osm.cskt0attuzeu.eu-west-1.rds.amazonaws.com:5432/osm?user=osm&password=X"`

We eventually hit the following exception:
```
Exception in thread "Thread-1" java.lang.RuntimeException: Thread managing SQL COPY to Postgres database failed.
	at com.conveyal.osmlib.PostgresSink.lambda$initiatePostgresCopy$0(PostgresSink.java:95)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.postgresql.util.PSQLException: ERROR: invalid byte sequence for encoding "UTF8": 0x81
  Where: COPY nodes, line 30273512: "660435579	49.6637968	10.957966	note=Bild vom Wegeingang;source_ref=C:\Users\Marcus\Pictures\Virb\201..."
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2412)
	at org.postgresql.core.v3.QueryExecutorImpl.processCopyResults(QueryExecutorImpl.java:1043)
	at org.postgresql.core.v3.QueryExecutorImpl.writeToCopy(QueryExecutorImpl.java:925)
	at org.postgresql.core.v3.CopyInImpl.writeToCopy(CopyInImpl.java:35)
	at org.postgresql.copy.CopyManager.copyIn(CopyManager.java:224)
	at com.conveyal.osmlib.PostgresSink.lambda$initiatePostgresCopy$0(PostgresSink.java:93)
	... 1 more
```
This is even with the PrintWriter set to UTF-8.

Existing Java objects which have the desired behavior of catching/correcting bad character sequences include:

```Java
final BufferedInputStream in = new BufferedInputStream(istream);
final Reader inputReader = new InputStreamReader(in, StandardCharsets.UTF_8);
```

and CharsetEncoder / CharsetDecoder, which has settable behavior when it hits bad characters.

**Comment by abyrd** [comment]
The reason for this was not some mysterious non-utf8-codable values in a String, but rather the presence of backslashes in the strings. Someone had put a Windows file path containing backslashes in an OSM tag key, and that path contained a directory name that was numeric. Postgres was seeing that in the COPY text format and taking it as an escape sequence for a character that doesn’t exist. The short term solution is to strip out all special values from our tags: tabs, CR, LF (the delimiters) and backslashes (the escape character). This is much simpler than trying to properly escape all these things.

**Comment by abyrd** [comment]
Fixed via 5e1c7b2222adaef1a1f8c56e9234ba8a2d48a1c8 and ccc7f46082889d8431638ff09b67190f626b9247
